src/components/CurriculumComponents/CurriculumDownloadTab/index.tsx

```diff
@@ -148,7 +211,14 @@ const CurriculumDownloadTab: FC<CurriculumDownloadTabProps> = ({
   return (
     <OakThemeProvider theme={oakDefaultTheme}>
       <Box $maxWidth={1280} $mh={"auto"} $ph={18} $pb={[48]} $width={"100%"}>
-        {!isLoading && (
+        {subjectTierSelectionVisible === true && (
```

Comment:
Minor: This is probably clearer if you use a "mode" or something like that. So `mode` would have two type `"tier_selection" | "download"` and then you'd render the children based off of that.

**Note**: No need to fix this now, but just for reference.